src/dbus/server/introspect.xml

```diff
@@ -27,6 +27,21 @@
       <arg name="scan_result" type="a(tstayqqqqyybb)" direction="out"/> 
     </method>
 
+    <!-- Energy Scan: Perform a Thread energy scan.
+      @result: array of energy scan results.
+
+      The result struture definition is:
+      <literallayout>
+        struct {
+          uint8  channel
+          int8_t max_rssi
+        }
+      </literallayout>
+    -->
+    <method name="EnergyScan">
```

Comment:
what channels to scan, preferred or  supported?

Reply:
As discussed offline, we will always scan the preferred channels.